openshift.spec

```diff
@@ -149,44 +223,66 @@ do
   echo "+++ INSTALLING ${bin}"
   install -p -m 755 _build/bin/${bin} %{buildroot}%{_bindir}/${bin}
 done
+
+# openshift == atomic-enterprise
+install -p -m 755 _build/bin/openshift _build/bin/atomic-enterprise
+install -p -m 755 _build/bin/atomic-enterprise %{buildroot}%{_bindir}/atomic-enterprise
+
 # Install 'openshift' as client executable for windows and mac
-install -p -m 755 _build/bin/openshift %{buildroot}%{_datadir}/%{name}/linux/oc
-install -p -m 755 _build/bin/darwin_amd64/openshift %{buildroot}%{_datadir}/%{name}/macosx/oc
-install -p -m 755 _build/bin/windows_386/openshift.exe %{buildroot}%{_datadir}/%{name}/windows/oc.exe
+for pkgname in openshift atomic-enterprise
+do
+  install -d %{buildroot}%{_datadir}/${pkgname}/{linux,macosx,windows}
+  install -p -m 755 _build/bin/openshift %{buildroot}%{_datadir}/${pkgname}/linux/oc
+  install -p -m 755 _build/bin/darwin_amd64/openshift %{buildroot}%{_datadir}/${pkgname}/macosx/oc
+  install -p -m 755 _build/bin/windows_386/openshift.exe %{buildroot}%{_datadir}/${pkgname}/windows/oc.exe
```

Comment:
for ctrlcmd in oc aec; do
        install -p -m 755 _build/bin/openshift %{buildroot}%{_datadir}/${pkgname}/linux/$ctrlcmd
        install -p -m 755 _build/bin/darwin_amd64/openshift %{buildroot}%{_datadir}/${pkgname}/macosx/$ctrlcmd
        install -p -m 755 _build/bin/windows_386/openshift.exe %{buildroot}%{_datadir}/${pkgname}/windows/$ctrlcmd.exe
    done